domains/client/domain-operator/src/tests.rs

```diff
@@ -1639,7 +1629,10 @@ async fn test_domain_tx_propagate() -> Result<(), tokio::time::error::Elapsed> {
 
     async {
         while alice.sync_service.is_major_syncing() || bob.sync_service.is_major_syncing() {
-            tokio::time::sleep(std::time::Duration::from_millis(100)).await;
+            tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+            // Unfortunately, this test contains a race condition where Alice sometimes bans Bob
```

Comment:
Should there be a TODO to fix this then?
Maybe related to https://github.com/paritytech/polkadot-sdk/issues/531?

Reply:
Sure!

```suggestion
            // TODO: Unfortunately, this test contains a race condition where Alice sometimes bans Bob
```